test/utils.jl

```diff
@@ -103,7 +114,6 @@ function temp_pkg_dir(fn::Function;rm=true)
             try
                 push!(LOAD_PATH, "@", "@v#.#", "@stdlib")
                 push!(DEPOT_PATH, depot_dir)
-                push!(DEPOT_PATH, REGISTRY_DEPOT)
```

Comment:
I'm not sure why this was included here. It seems to muddy up the state for the registry tests? My understanding may be limited though